src/LazyOperations/ExponentialMap.jl

```diff
@@ -60,6 +60,8 @@ struct SparseMatrixExp{N, MN<:AbstractSparseMatrix{N}} <: AbstractMatrix{N}
     M::MN
 end
 
+isoperation(::Type{SparseMatrixExp}) = true
```

Comment:
Was this on purpose? `SparseMatrixExp` is not actually a set (same for `ProjectionSparseMatrixExp`).

Reply:
true, this should be removed (same for the projection)